api/tests/acceptance/application/tags/tag-api_test.js

```diff
@@ -0,0 +1,63 @@
+const {
+  expect,
+  generateValidRequestAuthorizationHeader,
+  databaseBuilder,
+} = require('../../../test-helper');
+
+const createServer = require('../../../../server');
+
+describe('Acceptance | Route | tag-router', () => {
+
+  describe('GET /api/tags', () => {
+
+    it('should return a list of tags with 200 HTTP status code', async () => {
+      // given
+      const server = await createServer();
+      const tag1 = databaseBuilder.factory.buildTag({ id: 1, name: 'TAG1' });
+      const tag2 = databaseBuilder.factory.buildTag({ id: 2, name: 'TAG2' });
+      const tag3 = databaseBuilder.factory.buildTag({ id: 3, name: 'TAG3' });
+      const userId = databaseBuilder.factory.buildUser().id;
```

Comment:
Pour faire suite au commentaire sur l'ajout d'un pre-handler pour ne pas rendre accessible cette route à tous les utilisateurs authentifiés :

```suggestion
      const userId = databaseBuilder.factory.buildUser.withPixRolePixMaster().id;
```

Reply:
Done